.travis.yml

```diff
@@ -1,9 +1,23 @@
 language: ruby
```

Comment:
We'll probably want to add those comatibility changes (and others like the github actions additions 👍 ) to the CHANGELOG... The compat changes themselves I like ;-)

Reply:
Absolutely, I just wanted to get feedback on the direction first.